host.py

```diff
@@ -122,7 +123,47 @@ def _read_output(self, cmd: str, log_level: int = logging.DEBUG) -> Result:
 
         return Result(out, err, exit_code)
 
+    def remove(self, source):
+        if self.is_localhost():
+            if os.path.exists(source):
+                os.remove(source)
+        else:
+            try:
+                sftp = self._host.open_sftp()
+                sftp.remove(source)
+            except FileNotFoundError:
+                pass
+
+    def put_sftp(self, local_file, remote_file):
+        assert not self.is_localhost()
+        while True:
+            try:
+                sftp = self._host.open_sftp()
+                if not os.path.exists(local_file):
+                    logger.error(f"Can't tranfer missing local {local_file} to {self._hostname}")
+                    sys.exit(-1)
+                return sftp.put(local_file, remote_file)
+            except Exception as e:
+                logger.info(e)
+                logger.info(f"Connection lost while trying to open sftpd connection, reconnecting...")
+                self.ssh_connect_looped(self._username, self._password)
+
+    def copy_iso_on_remote(self, iso_path, image_path) -> str:
```

Comment:
`copy_iso_on_remote` This function looks like something that doesn't belong into host.py due to eh the assertion at the beginning of the function. I'd like to have it as a helper function.